cmd/dashboard.go

```diff
@@ -152,9 +152,10 @@ var DashboardCmd = &cobra.Command{
 }
 
 func init() {
-	DashboardCmd.Flags().BoolVarP(&kubernetesMode, "kubernetes", "k", false, "Start Dapr dashboard in local browser")
-	DashboardCmd.Flags().BoolVarP(&dashboardVersion, "version", "v", false, "Check Dapr dashboard version")
-	DashboardCmd.Flags().IntVarP(&dashboardLocalPort, "port", "p", defaultLocalPort, "The local port on which to serve dashboard")
+	DashboardCmd.Flags().BoolVarP(&kubernetesMode, "kubernetes", "k", false, "Starts proxy for Dapr dashboard on Kubernetes and opens local browser")
```

Comment:
Should this not be "Opens Dapr dashboard in local browser attached to Kubernetes cluster"

Reply:
The browser is not attached to K8s.